source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,181 @@
+# Install Prerequisites
+
+[//]: # (If any commented statements become false, change prior text)
+
+Before you begin, you will need to have access to our Beta. If you don't have access yet, request access at [bons.ai][1] and then come visit us in the [Forums][17] and let us know why you're excited to try out Bonsai!
+
+The next section walks you through how to install Python, Git, and Pip, but if you're already ahead of the curve and have them all installed, skip down to [Setup the Bonsai CLI][2]. 
+
+**Note:** We highly recommend Mac users use [HomeBrew][6] or download Python directly in the General Install instructions so they can perform the following steps on user-land Python rather than [system Python][13].
+
+**If you have Anaconda installed on your system, skip down to [Install with Anaconda][9].**
+
+If you *do not* have Anaconda installed on your system continue to follow our **General Install** instructions.
+
+## General Install
+
+### Windows
+
+> [Chocolatey][14] commands
+
+```shell
+choco install git
+choco install python # or you can install python2 if you need version 2.7
+```
+
+1. **Git**: Download and install Git from [Git for Windows][11] or with [Chocolatey][14].
+2. **Python**: Download and install Python from [python.org][3] or with [Chocolatey][14].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+**Note:** You may need to reboot your computer after installing Python and Git to make sure all install settings take effect.
+
+### Mac OS X
+
+> [Homebrew][6] commands
+
+```shell
+brew install git
+brew install python3 # or you can install python if you need version 2.7
+```
+
+1. **Git**: Git is more than likely already installed on your computer if you have Xcode, but if not, run the git command or install with [Homebrew][6].
+2. **Python**: Download and install Python from [python.org][3] or with [Homebrew][6].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+## Install with Anaconda
+
+If you *do not* have Anaconda installed on your system, please skip this section.
+
+conda is a tool for installing and managing Python and R dependencies. For more information, refer to the [Anaconda website][10].
+
+> Windows
+
+```shell
+conda create -n gym-env
+activate gym-env
+conda install pip
+```
+
+> Mac OS
+
+```shell
+conda create -n gym-env
+source activate gym-env
+conda install pip
+```
+
+# Setup the Bonsai CLI
+
+Install the Bonsai Command Line Interface tool and configure it. You'll need to have your access code (accessKey). You can find your access code in your [Bonsai Account Settings][8].
+
+```shell
+pip install bonsai-cli
```

Comment:
> srainier an hour ago Member
I want to point out something to be reeeeeally careful about in our documentation. One way (among many) to get screwed up with pip is to assume the literal pip command installs packages to the python you just installed. The only time I've found pip to always refer to the environment's currently active python is if you use a virtualenv (which these instructions, understandably, do not). A really common error is for a user to install python3 and then try to pip install ... packages, and those packages get installed into a different python2 site-packages. In this case the user should have used pip3 rather than pip. I've found that various ways of installing python and pip on the various platforms we support are pretty inconsistent with what pip maps to when python3 is installed. This isn't necessarily part of the changes here, but I saw it so I'm calling it out. If you want to take on a big amount of work to get this right, what you'll need to do is:

1. Get 'fresh' environments for both Mac and Windows with regard to python2/3 and pip being installed.
2. Separately (and independently) install both python2 and python3 via the suggested methods and write down what pip and pip3 (if python3 is involved) install to.
3. If you find that all means of installing python2 and python3 produce a pip that always installs to the user's intended python, you don't have to do anything more. If you find inconsistencies, you may need to produce separate instructions for python2 and python3, and maybe Windows and macOS.

I'm happy to elaborate on this as I have available time.

@keenbrowne thoughts on how to approach this?